src/tox/tox_env/api.py

```diff
@@ -132,7 +132,7 @@ def register_config(self) -> None:
 
         def pass_env_post_process(values: list[str]) -> list[str]:
             values.extend(self._default_pass_env())
-            return sorted({k: None for k in values}.keys())
```

Comment:
No operation might happen here because this is past the INI file loader: translation this will break a soon-to-add native `pyproject.toml` config intuitive behavior, which I'm not willing to put up with. You can't make changes to the INI file loader because that applies to all other keys, and e.g. for deps you don't want this. Hence my comment, this can't be done clearly, so I'm willing to make a breaking change.